sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/PageBlobAsyncClient.java

```diff
@@ -336,7 +337,7 @@
      * @return
      *      A reactive response containing the information of the cleared pages.
      */
-    public Flux<PageRange> getPageRanges(BlobRange blobRange) {
+    public Mono<Response<PageList>> getPageRanges(BlobRange blobRange) {
```

Comment:
We may want to undo these changes (and any others made to listing operation return types). The PagedFlux<T> changes are in PR, and this will just be one more conflict that we know the winner of already.

Reply:
Discussed to deal with merge.